admin/tabs/cluster/instances/websocket/WebSocketColumns.ts

```diff
@@ -72,3 +72,12 @@ export const lastReceivedTime: ColumnSpec = {
     ...Col.compactDate,
     width: 140
 };
+
+export const clientAppVersion: ColumnSpec = {
+    field: {
+        name: 'clientAppVersion',
+        type: 'string',
+        displayName: 'Client App Version'
+    },
+    width: 120
```

Comment:
120 is holding "7.0 SNAPSHOT" with no issue, but unsure how much we expect this value could grow.